src/tooltip/src/Tooltip.js

```diff
@@ -156,9 +156,14 @@ export default class Tooltip extends PureComponent {
     } = this.props
     const { isShown: stateIsShown, isShownByTarget } = this.state
 
-    const shown =
+    let shown =
       (isShown || stateIsShown || isShownByTarget) && !this.isPopoverShown()
```

Comment:
Wouldn't we also want to respect `this.props.isShown` here, regardless of `!this.isPopoverShown()`?

Reply:
I don't think I understand, @jeroenransijn definitely will though! Is this an error I can reproduce?